server/internal/handler/eatery.go

```diff
@@ -19,11 +20,29 @@ func (h *Handler) GetEateries(c echo.Context, params schema.GetEateriesParams) e
 }
 
 // PostEateries implements schema.ServerInterface.
-func (h *Handler) PostEateries(c echo.Context, params schema.PostEateriesParams) error {
-	return c.JSON(http.StatusNotImplemented, schema.Error{
-		Code:  "NOT_IMPLEMENTED",
-		Error: "PostEateries endpoint is not implemented yet",
-	})
+func (h *Handler) PostEateries(c echo.Context, params schema.GetEateriesParams) error {
```

Comment:
型が違うとビルドが通らなくなってしまうので`PostEateriesParams`でお願いします :pray:
```suggestion
func (h *Handler) PostEateries(c echo.Context, params schema.PostEateriesParams) error {
```